optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeUserTaskImportIT.java

```diff
@@ -41,7 +41,8 @@
 public class ZeebeUserTaskImportIT extends AbstractCCSMIT {
 
   private static final String TEST_PROCESS = "aProcess";
-  private static final String DUE_DATE = "2023-11-01T12:00:00+05:00";
+  private static final String DUE_DATE = "2024-07-24T00:00Z[GMT]";
+  private static final OffsetDateTime EXPECTED_DUE_DATE = OffsetDateTime.parse("2024-07-24T00:00Z");
```

Comment:
That would not be parseable, essentially that would recreate the bug this is fixing. Its also not using the helper to make sure we're specifically assessing this string as expected (so that eg if the parser helper had issues, we would see this here too)